recipes/netpbm/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "10.68" %}
+
+package:
+  name: netpbm
+  version: {{ version }}
+
+source:
+  # See http://netpbm.svn.sourceforge.net/viewvc/netpbm/
+  svn_url: http://svn.code.sf.net/p/netpbm/code/advanced
+  svn_rev: 2294
```

Comment:
Though I don't see why it is an issue to switch to the tarball. Could you please try, @183amir?